code/src/pages/cart/CartPage.scss

```diff
@@ -0,0 +1,78 @@
+@import "@design-system";
+
+.spa-cart-page {
+  padding: spacing(10);
+
+  &__content {
+    display: flex;
+    justify-content: space-between;
+    gap: spacing(5);
+  }
+
+  &__items,
+  &__order-summary {
+    padding: spacing(5);
+    border: 1px solid var(--spa-order-summary-border-color, $white);
+    border-radius: spacing(1);
+    background-color: var(--spa-order-summary-background-color, $white);
+  }
+
+  &__items {
+    width: 60%;
+
+    h1 {
+      margin-bottom: spacing(5);
+      font-size: spacing(8);
+      border-bottom: 1px solid
+        var(--spa-order-cart-title-border-color, $gray-light);
+      padding-bottom: spacing(3);
+    }
+  }
+
+  &__order-summary {
+    width: 35%;

```

Comment:
Also here. Please avoid % in width property

Reply:
> Also here. Please avoid % in width property

Why ?